cdc/redo/writer/mock_RedoLogWriter.go

```diff
@@ -1,17 +1,4 @@
-//  Copyright 2021 PingCAP, Inc.
```

Comment:
Should the copyright information be retained?

Reply:
This is a fully generated file like pb.go. So I think it is unnecessary.